common/xbps-src/shutils/common.sh

```diff
@@ -250,6 +250,17 @@ source_file() {
     fi
 }
 
+arrayise() {
+    for name; do
+        if [[ -n "$name" && "$(eval echo \${$name@a})" != *a* ]]; then
+            IFS=" " read -r -a "${name}" <<< "$(eval echo "\${$name}")"
+            if [ "$(eval echo \${#$name})" -gt 1 ]; then
```

Comment:
Here you need https://unix.stackexchange.com/questions/281390/how-to-get-the-size-of-an-indirect-array-in-bash